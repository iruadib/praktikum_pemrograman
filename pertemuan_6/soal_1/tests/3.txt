2
013
Kertas
1
1
014
Pensil
1
1