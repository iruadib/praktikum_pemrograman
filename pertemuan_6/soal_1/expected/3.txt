Jumlah barang: 2
Rata-rata harga barang: 1.00
